ui/js/filters.js

```diff
@@ -2,17 +2,12 @@
 /* eslint-disable func-names, prefer-arrow-callback */
 import numeral from 'numeral';
 
-import { getJobsUrl } from '../helpers/url';
+import { getRevisionUrl } from '../perfherder/helpers';
 
 import treeherder from './treeherder';
 
 treeherder.filter('getRevisionUrl', function () {
-    return function (revision, projectName) {
-        if (revision) {
-            return getJobsUrl({ repo: projectName, revision });
-        }
-        return '';
-    };
+    return getRevisionUrl;
```

Comment:
Can make this a one-liner with the arrow function.

Reply:
Or even:

```js
treeherder.filter('getRevisionUrl', getRevisionUrl);
```